itez/beneficiary/tests/test_utils.py

```diff
@@ -0,0 +1,37 @@
+from django.test import TestCase
+
+from itez.beneficiary.utils import generate_uuid_and_agent_code
+
+
+class UUIDAndAgentCode(TestCase):
+
+    def setUp(self):
+        """
+         This method is called for every test that runs and 
+         helps with variable declaration
+        """
+        self.uuid_code = generate_uuid_and_agent_code()[0]
+        self.agent_code = generate_uuid_and_agent_code()[1]
+
+    def tearDown(self):
+        """
+         This method is called for every test as well
+        """
+
+    def test_uuid_valid(self):
+        """
+         This method tests for the validity of generated uuid.
+        """
+        self.assertEqual(len(str(self.uuid_code)), 36)
+
+    def test_agent_code_length(self):
+        """
+         This method tests for length of agent code.
```

Comment:
full stop at end of line

Reply:
- [x] add full stop + screenshot( ' passing tests ' )

![working test](https://user-images.githubusercontent.com/48860145/140495682-49f18690-0d67-41d1-809e-06df6fd50830.png)